tests/e2e/k8s/snapshotter.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nydus-system
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nydus-snapshotter-sa
+  namespace: nydus-system
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nydus-snapshotter-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nydus-snapshotter-role-binding
+roleRef:
+  kind: ClusterRole
+  name: nydus-snapshotter-role
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: nydus-snapshotter-sa
+  namespace: nydus-system
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nydus-snapshotter
+  namespace: nydus-system
+spec:
+  containers:
+  - env:
+    - name: ENABLE_NYDUS_OVERLAY
+      value: "false"
+    name: nydus-snapshotter
+    image: local-dev:e2e
+    imagePullPolicy: IfNotPresent
+    args:
+      - containerd-nydus-grpc
+      - --nydusd-path /usr/local/bin/nydusd
+      - --config-path /etc/nydus/config.json
+      - --root /var/lib/containerd-nydus
+      - --address /run/containerd-nydus/containerd-nydus-grpc.sock
+      - --log-level debug
+      - --enable-nydus-overlayfs=false
```

Comment:
Where do we pack the binary `nydus-overlayfs` ?

Reply:
removed